timon/db/backends.py

```diff
@@ -121,6 +149,18 @@ async def store_probe_result(self, probename, timestamp, msg, status):
             await trio.sleep(0.1)
         self.storersltqueue.put(prb_rslt)
 
+    async def store_hist_probe_result(self, probename, timestamp, msg, status):
+        prb_rslt = {
+            "name": probename,
+            "msg": msg,
+            "status": status,
+            "dt": datetime.fromtimestamp(timestamp),
+        }
+        while self.storehistrsltqueue.full():
```

Comment:
perhaps add a log? (at least the first time)
and perhaps a log when one had to wait and can finally add to queue.
Might help discovering blocks.

can close

Reply:
done